auditlog/tests/test_auditlog.py

```diff
@@ -259,6 +259,28 @@ def test_LogDelete(self):
             1,
         )
 
+    def test_LogName_when_record_unlinked_twice(self):
```

Comment:
Mixed camel casing  and snake style. Please rename to `test_log_name_when_unlinked_twice`.